models/public/common-sign-language-0001/common-sign-language-0001.md

```diff
@@ -0,0 +1,73 @@
+# common-sign-language-0001
+
+## Use Case and High-Level Description
+
+A human gesture recognition model for the Jester dataset recognition scenario
+(word-level recognition). The model uses an S3D framework with MobileNet V3 backbone. Please refer
+to the [Jester\*](https://20bn.com/datasets/jester) dataset specification
+to see the list of gestures that are recognized by this model.
+
+The model accepts a stack of frames sampled with a constant framerate (15 FPS) and produces a prediction
+on the input clip.
+
+## Specification
+
+| Metric            | Value        |
+|-------------------|--------------|
+| Top-1 accuracy    | TBD          |
+| GFlops            | TBD          |
+| MParams           | TBD          |
+| Source framework  | PyTorch\*    |
+
+## Performance
+
+## Input
+
+### Original Model
+
+Batch of images of the shape [1x3x8x224x224] in the [BxCxTxHxW] format, where:
+
+- `B` - batch size
+- `C` - channel
+- `T` - sequence length
+- `H` - height
+- `W` - width
+
+Channel order is `RGB`.
+
+### Converted Model
+
+Batch of images of the shape [1x3x8x224x224] in the [BxCxTxHxW] format, where:
+
+- `B` - batch size
+- `C` - channel
+- `T` - sequence length
+- `H` - height
+- `W` - width
+
+Channel order is `RGB`.
```

Comment:
is it impossible to convert network with bgr channels order?
Can we use --reverse_input_channels option during MO conversion?

Reply:
Unfortunately MO supports `--reverse_input_channels` parameter for 4-dimensional input only but the network input is 5-dimensional. So, the answer is no.